### DISCUSSION TITLE
Add `-oldest` label

### DISCUSSION THREAD

**Discussion by kasper9n** [actions/virtual-environments#9353] [Ideas]
When building desktop apps on older OS versions, it's good to build using an older OS version as that improves your app's compatibility. Having `-oldest` to automatically build on the oldest supported runner would be huge.

I have a bunch of apps, and it's annoying to have to periodically update the `platform` label over and over for each app when old OS versions are dropped.

Related:
- https://github.com/actions/runner-images/issues/6354 (different use case, and closed due to misunderstanding)

**Comment by mikhailkoliada** [comment]
Hello! The answer @samvantran [posted](https://github.com/actions/runner-images/issues/6354#issuecomment-1270706679) is still valid, it still can not be implemented at the moment, unfortunately

**Reply by kasper9n** [reply]
Could you elaborate on why? That answer talks about image deprecation awareness, but that isn't really relevant (the whole issue is having to update many workflow files, awareness doesn't change that). And it doesn't mention why it cannot be implemented
